components/dialog/stories/dialog.test.js

```diff
@@ -51,39 +51,56 @@ export const DialogGroup = Variants({
 	],
 });
 
-export const DialogFullscreen = Variants({
-	Template: (args, context) => {
-		const { parameters: { showTestingGrid = false } = {} } = context;
+export const DialogFullscreen = (args, context) => {
+	const { parameters: { showTestingGrid = false } = {} } = context;
 
-		return Template({
-			...args,
-			/*
-			 * Custom styles were added to make sure the VRTs catch the rounded corners
-			 */
-			customStyles: {
-				margin: showTestingGrid ? "16px" : undefined,
-			},
-		}, context);
-	},
-	withSizes: false,
-	wrapperStyles: {
-		"background-color": "var(--spectrum-gray-50)"
-	},
-	testData: [
-		{
-			showModal: false,
-			layout: "fullscreen",
+	return Variants({
+		Template: (args, context) => {
+			return Template({
+				...args,
+				/*
+				 * Custom styles were added to make sure the VRTs catch the rounded corners
+				 */
+				customStyles: {
+					margin: showTestingGrid ? "16px" : undefined,
+				},
+			}, context);
 		},
-	],
-});
+		withSizes: false,
+		wrapperStyles: {
+			"background-color": "var(--spectrum-gray-50)",
+			...(showTestingGrid && { "inline-size": "100%" }), // Applies conditional styles based on showTestingGrid- forces the data-inner-container to be full width so the container query doesn't break.
+		},
+		containerStyles: {
+			...(showTestingGrid && { "inline-size": "100%" }), // forces the data-outer-container to be full width so the container query doesn't break.
```

Comment:
@rise-erpelding What do you think about this approach instead? I didn't notice the horizontal scroll before, but I believe it's gone now everywhere. Let me know!